src/api/walker.ts

```diff
@@ -118,7 +118,7 @@ export class Walker<TOutput extends Output> {
         this.resolveSymlink(path, this.state, (stat, resolvedPath) => {
           if (stat.isDirectory()) {
             resolvedPath = normalizePath(resolvedPath, this.state.options);
-            if (exclude && exclude(entry.name, useRealPaths ? resolvedPath : path + pathSeparator)) return;
+            if (exclude && exclude(entry.name,useRealPaths ? resolvedPath : path + pathSeparator)) return;
```

Comment:
```suggestion
            if (exclude && exclude(entry.name, useRealPaths ? resolvedPath : path + pathSeparator)) return;
```
you missed this formatting change :P